### PR TITLE
Make final changes for 1.0

### PR DESCRIPTION
**Pull request by BenjaminPelletier**
This PR mostly reformats the YAML for appropriate and consistent whitespace, but creates an enormous diff while doing that.

In addition, a few small issues found during final review are fixed.

### CODE REVIEW

File: utm.yaml [hunk 1]
```diff
@@ -64,89 +62,111 @@ components:
...
+        received.
 
-        When using the `aud` claim to protect against the reuse of received tokens, and absent guidance on behalf of the competent authority to the contrary, the JWT `aud` claim requested by the client must be included in each access token and must contain the fully qualified domain name of the URL the access token will be used to access.  For example, if a USS were querying the endpoint at https://dss.example.com:8888/rid/v2/dss/identification_service_areas, the access token included in the request should specify `"aud": "dss.example.com"`.
+        * `exp`, with a time no further than 1 hour in the future.
 
-        Clients must provide these access tokens in an `Authorization` header in the form `Bearer <token>` in accordance with RFC 6750.
+        * `sub`, with unique ID of the client requesting the access token.
 
+        * `scope`, with a string composed of a space-separated list of strings
+        indicating the scopes granted, per RFC 6749. * `jti`, according to RFC
```
Comment (davidmurphy1152) [inline utm.yaml:97]: Need line break.

File: utm.yaml [hunk 2]
```diff
@@ -2146,45 +2250,50 @@ paths:
...
         '400':
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * One or more input parameters were missing or invalid.
-            * The request attempted to mutate the operational intent in a disallowed way.
+          description: >-
+            * One or more input parameters were missing or invalid. * The
```
Comment (davidmurphy1152) [inline utm.yaml:2264]: Need line break

File: utm.yaml [hunk 3]
```diff
@@ -2146,45 +2250,50 @@ paths:
...
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/AirspaceConflictResponse'
-          description: |-
-            * The provided key did not prove knowledge of all current and relevant airspace Entities
-            * Despite repeated attempts, the DSS was unable to complete the update because of other simultaneous changes.
+          description: >-
+            * The provided key did not prove knowledge of all current and
+            relevant airspace Entities * Despite repeated attempts, the DSS was
```
Comment (davidmurphy1152) [inline utm.yaml:2290]: Need line break

File: utm.yaml [hunk 4]
```diff
@@ -2240,46 +2350,52 @@ paths:
...
         '400':
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * One or more input parameters were missing or invalid.
-            * The request attempted to mutate the operational intent in a disallowed way.
+          description: >-
+            * One or more input parameters were missing or invalid. * The
```
Comment (davidmurphy1152) [inline utm.yaml:2364]: Need line break

File: utm.yaml [hunk 5]
```diff
@@ -2240,46 +2350,52 @@ paths:
...
             application/json:
               schema:
                 $ref: '#/components/schemas/AirspaceConflictResponse'
-          description: |-
-            * The provided key did not prove knowledge of all current and relevant airspace Entities
-            * The provided `ovn` does not match the current version of the existing operational intent.
-            * Despite repeated attempts, the DSS was unable to complete the update because of other simultaneous changes.
+          description: >-
+            * The provided key did not prove knowledge of all current and
+            relevant airspace Entities * The provided `ovn` does not match the
```
Comment (davidmurphy1152) [inline utm.yaml:2390]: Need line break

File: utm.yaml [hunk 6]
```diff
@@ -2240,46 +2350,52 @@ paths:
...
+            current version of the existing operational intent. * Despite
```
Comment (davidmurphy1152) [inline utm.yaml:2391]: Need line break

File: utm.yaml [hunk 7]
```diff
@@ -2345,31 +2465,34 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * The provided `ovn` does not match the current version of the existing operational intent.
-            * Despite repeated attempts, the DSS was unable to complete the update because of other simultaneous changes.
+          description: >-
+            * The provided `ovn` does not match the current version of the
+            existing operational intent. * Despite repeated attempts, the DSS
```
Comment (davidmurphy1152) [inline utm.yaml:2473]: Need line break

File: utm.yaml [hunk 8]
```diff
@@ -2517,37 +2645,41 @@ paths:
...
         '400':
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * One or more input parameters were missing or invalid.
-            * The request attempted to mutate the constraint in a disallowed way.
+          description: >-
+            * One or more input parameters were missing or invalid. * The
```
Comment (davidmurphy1152) [inline utm.yaml:2659]: Need line break

File: utm.yaml [hunk 9]
```diff
@@ -2517,37 +2645,41 @@ paths:
...
         '409':
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * A constraint with the provided ID already exists.
-            * Despite repeated attempts, the DSS was unable to complete the update because of other simultaneous changes.
+          description: >-
+            * A constraint with the provided ID already exists. * Despite
```
Comment (davidmurphy1152) [inline utm.yaml:2683]: Need line break

File: utm.yaml [hunk 10]
```diff
@@ -2598,37 +2731,41 @@ paths:
...
         '400':
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * One or more input parameters were missing or invalid.
-            * The request attempted to mutate the constraint in a disallowed way.
+          description: >-
+            * One or more input parameters were missing or invalid. * The
```
Comment (davidmurphy1152) [inline utm.yaml:2745]: Need line break

File: utm.yaml [hunk 11]
```diff
@@ -2598,37 +2731,41 @@ paths:
...
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * The provided `ovn` does not match the current version of the existing constraint.
-            * Despite repeated attempts, the DSS was unable to complete the update because of other simultaneous changes.
+          description: >-
+            * The provided `ovn` does not match the current version of the
+            existing constraint. * Despite repeated attempts, the DSS was unable
```
Comment (davidmurphy1152) [inline utm.yaml:2770]: Need line break

File: utm.yaml [hunk 12]
```diff
@@ -2663,23 +2800,25 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * One or more input parameters were missing or invalid.
-            * The request attempted to mutate the constraint in a disallowed way.
+          description: >-
+            * One or more input parameters were missing or invalid. * The
```
Comment (davidmurphy1152) [inline utm.yaml:2807]: Need line break

File: utm.yaml [hunk 13]
```diff
@@ -2691,23 +2830,23 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * The provided `ovn` does not match the current version of the existing constraint.
-            * Despite repeated attempts, the DSS was unable to complete the update because of other simultaneous changes.
+          description: >-
+            * The provided `ovn` does not match the current version of the
+            existing constraint. * Despite repeated attempts, the DSS was unable
```
Comment (davidmurphy1152) [inline utm.yaml:2838]: Need line break

File: utm.yaml [hunk 14]
```diff
@@ -2874,40 +3022,44 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * One or more input parameters were missing or invalid.
-            * The request attempted to mutate the subscription in a disallowed way.
+          description: >-
+            * One or more input parameters were missing or invalid. * The
```
Comment (davidmurphy1152) [inline utm.yaml:3029]: Need line break

File: utm.yaml [hunk 15]
```diff
@@ -2874,40 +3022,44 @@ paths:
...
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * The access token was decoded successfully but did not include a scope appropriate to this endpoint or the request.
-            * Client attempted to request notifications for an Entity type to which the scopes included in the access token do not provide access.
+          description: >-
+            * The access token was decoded successfully but did not include a
+            scope appropriate to this endpoint or the request. * Client
```
Comment (davidmurphy1152) [inline utm.yaml:3046]: Need line break

File: utm.yaml [hunk 16]
```diff
@@ -2874,40 +3022,44 @@ paths:
...
         '409':
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * A subscription with the specified ID already exists.
-            * Despite repeated attempts, the DSS was unable to complete the update because of other simultaneous changes.
+          description: >-
+            * A subscription with the specified ID already exists. * Despite
```
Comment (davidmurphy1152) [inline utm.yaml:3055]: Need line break

File: utm.yaml [hunk 17]
```diff
@@ -2959,41 +3118,45 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * One or more input parameters were missing or invalid.
-            * The request attempted to mutate the subscription in a disallowed way.
+          description: >-
+            * One or more input parameters were missing or invalid. * The
```
Comment (davidmurphy1152) [inline utm.yaml:3125]: Need line break

File: utm.yaml [hunk 18]
```diff
@@ -2959,41 +3118,45 @@ paths:
...
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * The access token was decoded successfully but did not include a scope appropriate to this endpoint or the request.
-            * Client attempted to request notifications for an Entity type to which the scopes included in the access token do not provide access.
+          description: >-
+            * The access token was decoded successfully but did not include a
+            scope appropriate to this endpoint or the request. * Client
```
Comment (davidmurphy1152) [inline utm.yaml:3142]: Need line break

File: utm.yaml [hunk 19]
```diff
@@ -2959,41 +3118,45 @@ paths:
...
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * A subscription with the specified ID already exists and is managed by a different client.
-            * The provided `version` does not match the current subscription.
-            * Despite repeated attempts, the DSS was unable to complete the update because of other simultaneous changes.
+          description: >-
+            * A subscription with the specified ID already exists and is managed
+            by a different client. * The provided `version` does not match the
```
Comment (davidmurphy1152) [inline utm.yaml:3152]: Need line break

File: utm.yaml [hunk 20]
```diff
@@ -2959,41 +3118,45 @@ paths:
...
+            current subscription. * Despite repeated attempts, the DSS was
```
Comment (davidmurphy1152) [inline utm.yaml:3153]: Need line break

File: utm.yaml [hunk 21]
```diff
@@ -3045,24 +3213,24 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * A subscription with the specified ID is managed by a different client.
-            * The provided `version` does not match the current subscription.
-            * Despite repeated attempts, the DSS was unable to complete the deletion because of other simultaneous changes.
+          description: >-
+            * A subscription with the specified ID is managed by a different
+            client. * The provided `version` does not match the current
```
Comment (davidmurphy1152) [inline utm.yaml:3221]: Need line break

File: utm.yaml [hunk 22]
```diff
@@ -3045,24 +3213,24 @@ paths:
...
+            subscription. * Despite repeated attempts, the DSS was unable to
```
Comment (davidmurphy1152) [inline utm.yaml:3222]: Need line break

File: utm.yaml [hunk 23]
```diff
@@ -3099,45 +3268,47 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * One or more parameters were missing or invalid.
-            * The report could not be parsed, or contains unrecognized data.
+          description: >-
+            * One or more parameters were missing or invalid. * The report could
```
Comment (davidmurphy1152) [inline utm.yaml:3275]: Need line break

File: utm.yaml [hunk 24]
```diff
@@ -3402,51 +3585,55 @@ paths:
...
         '400':
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * One or more parameters were missing or invalid.
-            * The Entity could not be parsed, or contains illegal data.
+          description: >-
+            * One or more parameters were missing or invalid. * The Entity could
```
Comment (davidmurphy1152) [inline utm.yaml:3600]: Need line break

File: utm.yaml [hunk 25]
```diff
@@ -3402,51 +3585,55 @@ paths:
...
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * The access token was decoded successfully but did not include a scope appropriate to this endpoint.
-            * The client identified in the access token is not the manager of this Entity according to the receiving client's records.
+          description: >-
+            * The access token was decoded successfully but did not include a
+            scope appropriate to this endpoint. * The client identified in the
```
Comment (davidmurphy1152) [inline utm.yaml:3617]: Need line break

File: utm.yaml [hunk 26]
```diff
@@ -3535,52 +3723,56 @@ paths:
...
         '400':
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * One or more parameters were missing or invalid.
-            * The Entity could not be parsed, or contains illegal data.
+          description: >-
+            * One or more parameters were missing or invalid. * The Entity could
```
Comment (davidmurphy1152) [inline utm.yaml:3737]: Need line break

File: utm.yaml [hunk 27]
```diff
@@ -3535,52 +3723,56 @@ paths:
...
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * The access token was decoded successfully but did not include a scope appropriate to this endpoint.
-            * The client identified in the access token is not the manager of this Entity according to the receiving client's records.
+          description: >-
+            * The access token was decoded successfully but did not include a
+            scope appropriate to this endpoint. * The client identified in the
```
Comment (davidmurphy1152) [inline utm.yaml:3754]: Need line break

File: utm.yaml [hunk 28]
```diff
@@ -3619,41 +3813,41 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-          description: |-
-            * One or more parameters were missing or invalid.
-            * The report could not be parsed, or contains unrecognized data.
+          description: >-
+            * One or more parameters were missing or invalid. * The report could
```
Comment (davidmurphy1152) [inline utm.yaml:3820]: Need line break

File: utm.yaml [hunk 29]
```diff
@@ -1278,30 +1327,33 @@ components:
...
+      description: >-
+        A USS is presumed to be in the Unknown state absent indication otherwise
+        by a USS with availability arbitration scope.  Upon determination via
+        availability arbitration, a USS is Down when it does not respond
+        appropriately, and a Down USS may not perform the following operations
+        in the DSS:
+
+        * Create an operational intent in the Accepted or Activated states
 
-          * Create an operational intent in the Accepted or Activated states
```
[the remark this answers is not in the document]
Comment (BenjaminPelletier) [reply utm.yaml:1298]: These bullets look good in ReDoc with the current indentation:
image

Seems ok in SwaggerHub also:
image

[the remark this answers is not in the document]
Comment (davidmurphy1152) [reply utm.yaml:1298]: Here it is on swagger editor. I know we can't accommodate every rendering tool. Just another data point to consider.

image

**Pull request merged**